src/meshdb/celery.py

```diff
@@ -64,7 +65,7 @@ def beat_ready(**_: dict) -> None:
 os.environ.setdefault("DJANGO_SETTINGS_MODULE", "meshdb.settings")
 
 # Use the docker-hosted Redis container as the backend for Celery
-app = Celery("meshdb", broker=os.environ.get("CELERY_BROKER", "redis://localhost:6379/0"))
+app = Celery("meshdb", broker= environment.CELERY_BROKER)
```

Comment:
You should just be able to refer to this by the environment name, since you're importing it directly.

```suggestion
app = Celery("meshdb", broker=CELERY_BROKER)
```